isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend/spring.adoc

```diff
@@ -0,0 +1,290 @@
+= Verwendung von Spring
+
+include::documentation-guide::partial$licence.adoc[]
+
+Das Spring-Framework ist ein Java EE Framework, welches im Kern sehr verständlich und leicht zu verwenden ist.
+In ihm werden die Bestandteile eines Systems als _Spring-Beans_ definiert.
+Neben seiner Kern-Funktionalität der Verwaltung, Konfiguration und aspektorientierten Erweiterung von Beans bietet Spring viele Funktionalitäten, welche die Entwicklung einer Anwendung erleichtern sollen.
+
+Die folgenden Inhalte beschreiben die Nutzung von Spring für die allgemeine Umsetzung von Backends.
+Die Nutzung spezifischer Features von Spring sind Bestandteile der folgenden Bausteine:
+
+* Konfiguration der Persistenzschicht: beschrieben in den xref:isy-persistence:nutzungsvorgaben/umsetzung-datenzugriff.adoc[Nutzungsvorgaben JPA/Hibernate],
+* Konfiguration des Logging: beschrieben in den xref:isy-logging:nutzungsvorgaben/master.adoc[],
+* Bereitstellung von Metriken zur Überwachung: beschrieben im xref:isy-ueberwachung:konzept/master.adoc[].
+
+Alle anderen Funktionalitäten von Spring (Validierung über Spring, Emailing, Thread Pooling, Scripting) werden nicht verwendet.
+
+Mit der Verwendung von Spring verspricht man sich eine geringere Komplexität sowie einen geringeren Entwicklungsaufwand von Backends.
+Die Verwendung unterliegt den folgenden Zielen.
+
+Einheitlichkeit der Nutzung::
+Spring soll in Backends einheitlich eingesetzt werden.
+Hierfür sind geeignete Vorgaben für die Nutzung zu etablieren.
+
+Verständlichkeit der Konfiguration::
+Die Konfiguration der Spring-Komponenten erfolgt über Annotationen an den Komponenten sowie über Konfigurationsklassen.
+
+Komponentenorientierung wahren::
+Über Spring werden Komponenten konfiguriert.
+Es soll nicht möglich sein, direkt auf Implementierungsklassen einer Komponente zuzugreifen.
+
+[[konfiguration-von-spring]]
+== Konfiguration von Spring
+
+Das grundlegende Konzept von Spring ist die _Spring-Bean_.
+Die Konfiguration von Spring teilt sich deshalb in zwei Teile: in die Konfiguration der Spring-Beans sowie in die Konfiguration von Spring selbst (innerhalb eines Tomcat Servers).
+Um die Konfiguration von Spring und der eingesetzten Bausteine einfach zu halten, werden die Autokonfigurationsmechanismen von Spring Boot eingesetzt.
+Diese Konfigurationsarten werden in den folgenden Kapiteln beschrieben.
```

Comment:
Welche Konfigurationsarten sind hier gemeint? 
und in welchen Kapiteln werden sie beschrieben (das unmittelbar nachfolgende Kapitel beschreibt die Konfiguration von Spring Beans), weitere Kapitel, die ggf. die Konfiguration von Spring selbst beschreiben, befinden sich wo?

Reply:
Satz entfernt.